my-groups.js

```diff
@@ -44,7 +44,7 @@ async function LoadGroups(){
     const json = await response.json();
  
      //this line below is used for testing
```

Comment:
If you'd like, you could delete this line because it seems like it was for testing purposes. However, if you left it in for future testing then it's okay.